ArtOfIllusion/src/artofillusion/LayoutWindow.java

```diff
@@ -15,28 +15,39 @@ Changes copyright (C) 2017 by Petri Ihalainen
 import artofillusion.animation.*;
 import artofillusion.animation.distortion.*;
 import artofillusion.image.*;
+import artofillusion.keystroke.*;
 import artofillusion.math.*;
 import artofillusion.object.*;
 import artofillusion.script.*;
-import artofillusion.texture.*;
+import artofillusion.texture.Texture;
 import artofillusion.ui.*;
 import artofillusion.view.ViewAnimation;
-import artofillusion.keystroke.*;
 import buoy.event.*;
 import buoy.widget.*;
-
-import java.awt.*;
+import buoyx.docking.*;
+import java.awt.Color;
```

Comment:
Please don't expand package imports like this. I'm aware of the arguments in favor of explicit per-class imports, and in this context, I disagree with them.

```suggestion
import java.awt.*;
```